rke/k8s_rke_system_images.go

```diff
@@ -5258,6 +5258,49 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled in v2.5.10
```

Comment:
wait, is this not going out of band for v2.5.x?

Reply:
Right, let me update those.